internal/pkg/skuba/upgrade/node/versions.go

```diff
@@ -95,6 +97,16 @@ func (nviu NodeVersionInfoUpdate) IsFirstControlPlaneNodeToBeUpgraded(client cli
 // If all preconditions are met for the given node, no error will be returned.
 func (nviu NodeVersionInfoUpdate) NodeUpgradeableCheck(client clientset.Interface, currentClusterVersion *version.Version) error {
 	errorMessages := []string{}
+	if strings.HasPrefix(nviu.Update.KubeletVersion.String(), skubaconstants.LastCaaSP4KubernetesVersion) {
+		errorMessages = append(errorMessages, fmt.Sprintf("Cannot use this version of skuba to upgrade to kubernetes version %s. Please use a SP1 skuba binary instead.", skubaconstants.LastCaaSP4KubernetesVersion))
```

Comment:
I don't think the user will understand with `SP1 skuba`, maybe provide a package version? Or `skuba with version < X.X.X`

Reply:
Good idea! What does this map to? Do you know by any chance?

Reply:
According to this https://build.suse.de/package/view_file/SUSE:SLE-15-SP2:Update:Products:CaaSP:4.5/skuba/skuba.spec?expand=1 it should be `< 2.0.0`

Reply:
the last release seems to be 2.1.4 or something, so Yes it might be wise to have 2.0.0.